packages/react-native-gesture-handler/src/components/Pressable/PressableProps.tsx

```diff
@@ -171,6 +171,8 @@ export interface PressableProps
    * Defines the dimensions of the Pressable after it's been resized.
    * This property does not affect Pressable's physical appearance.
    * Required when the Pressable is resized **and** uses pressRetentionOffset.
+   *
+   * @deprecated This property is no longer used. It can be safely removed.
```

Comment:
Do we plan to remove it? If so, let's state it right away as this is fairly new prop and I guess most of users have not adapted it yet. Otherwise we could add entry in `docs`.